src/main/java/bio/terra/service/filedata/azure/util/AzureBlobStoreBufferedReader.java

```diff
@@ -0,0 +1,16 @@
+package bio.terra.service.filedata.azure.util;
+
+import com.azure.storage.blob.BlobClientBuilder;
+import java.io.BufferedReader;
+import java.io.InputStreamReader;
+import java.nio.charset.StandardCharsets;
+
+/** Given a gs path and a storage object, return a buffered reader for the blob */
```

Comment:
Looks good! but it's not a gs path :)
```suggestion
/** Given a signed azure path, return a buffered reader for the blob */
```

It sure is confusing that both azure and gcs call them blobs..

Reply:
Gah, my copy and paste has been uncovered haha